mono/metadata/sre.c

```diff
@@ -96,15 +96,11 @@ mono_reflection_emit_init (void)
 }
 
 char*
-string_to_utf8_image_raw (MonoImage *image, MonoString *s_raw, MonoError *error)
+string_to_utf8_image_raw (MonoImage *image, MonoPtr<MonoString>& s, MonoError *error)
 {
-	/* FIXME all callers to string_to_utf8_image_raw should use handles */
 	HANDLE_FUNCTION_ENTER ();
-	char* result = NULL;
 	error_init (error);
-	MONO_HANDLE_DCL (MonoString, s);
-	result = mono_string_to_utf8_image (image, s, error);
-	HANDLE_FUNCTION_RETURN_VAL (result);
+	return mono_string_to_utf8_image (image, s.NewHandle (), error);
```

Comment:
It counts for something when a function drops from 6 lines to 3, with no loss of clarity. It is still 3 too many though.